otj-metrics-reactive/src/main/java/com/opentable/metrics/reactive/MetricsHttpEndpoint.java

```diff
@@ -0,0 +1,94 @@
+package com.opentable.metrics.reactive;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.codahale.metrics.Counter;
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.Histogram;
+import com.codahale.metrics.Meter;
+import com.codahale.metrics.Metric;
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import com.codahale.metrics.health.HealthCheck;
+import com.codahale.metrics.health.HealthCheckRegistry;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import reactor.core.publisher.Mono;
+
+import com.opentable.metrics.http.CounterResponse;
+import com.opentable.metrics.http.GaugeResponse;
+import com.opentable.metrics.http.HealthCheckResponse;
+import com.opentable.metrics.http.HistogramResponse;
+import com.opentable.metrics.http.MeterResponse;
+import com.opentable.metrics.http.MonitorResponse;
+import com.opentable.metrics.http.TimerResponse;
+
+/**
+ * Metrics endpoint for Reactive HTTP servers.
+ */
+@RestController
+@RequestMapping("/service-status")
+public class MetricsHttpEndpoint {
+
+    private final MetricRegistry metrics;
+    private final HealthCheckRegistry health;
+
+    @Autowired
+    public MetricsHttpEndpoint(MetricRegistry metrics, HealthCheckRegistry health) {
+        this.metrics = metrics;
+        this.health = health;
+    }
+
+    @GetMapping
+    public Mono<List<MonitorResponse>> get()
+    {
+        final List<MonitorResponse> responses = new ArrayList<>();
+        metrics.getMetrics().forEach((n, m) -> responses.add(toResponse(n, m)));
+        health.runHealthChecks().forEach((n, r) -> responses.add(toResponse(n, r)));
```

Comment:
We could use https://metrics.dropwizard.io/3.1.0/apidocs/com/codahale/metrics/health/HealthCheckRegistry.html#runHealthChecks-java.util.concurrent.ExecutorService- to pass an executor to run the health checks in parallel, but that's probably a waste since we rarely use many health checks or expensive ones.